### PR TITLE
Relax ES checks in `/api/v1/probes/startup` REST Endpoint

### LINKED ISSUE #29719: relax ES checks in /api/v1/probes/startup
### Problem Statement

`/api/v1/probes/startup` returns 503 in situations when it should not. We are not sure of the details. It seems to occur if there is not an Active index, and possibly if the Active index is empty.

We use this for the `startupProbe` in kubernetes to determine when a new pod is healthy. Until it succeeds, the new pod is not fully in service. If the server is running a long-running reindex, the pod will be terminated by kubernetes while the reindex is in process, as the reindex time will exceed the default startup timeout we set for this probe. On a cluster environment, the first pod needs to be healthy before k8s will start other pods in the cluster.

### Steps to Reproduce

Delete all indexes, restart dotCMS. `/api/v1/probes/startup` will return 503 until there is an Active index. Something like that anyway.

### Acceptance Criteria

Not sure. Perhaps `/api/v1/probes/startup` can verify that Elasticsearch is reachable, without checking on the health of the Active index.

### dotCMS Version

LTS and current

### Proposed Objective

Reliability

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

https://dotcms.slack.com/archives/C072VF6R2JC/p1724294214346659

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

Postman tests are located in: `core/dotcms-postman/src/main/resources/postman/MonitorResource.postman_collection.json`

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by jcastro-dotcms**
### Proposed Changes
* Relax ES checks in `/api/v1/probes/startup` REST Endpoint in order to prevent 503 errors when there's simply an empty or inactive ES Index.
* Update the Postman Tests to reflect the new code changes.

This PR fixes: #29719

### CODE REVIEW

**Review by victoralfaro-dotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java [hunk 1]
```diff
@@ -21,104 +17,127 @@
...
-            if (ACLS_IPS == null || ACLS_IPS.length == 0) {
-                this.accessGranted = true;
-            } else {
-                for (String aclIP : ACLS_IPS) {
-                    if (IPUtils.isIpInCIDR(clientIP, aclIP)) {
-                        this.accessGranted = true;
-                        break;
-                    }
+
+            for (String aclIP : ACLS_IPS) {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java:84]: set to final

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java [hunk 2]
```diff
@@ -127,84 +146,73 @@ synchronized MonitorStats getMonitorStatsNoCache()  {
...
-        return totalHits > 0;
-
+    /**
+     * Determines if dotCMS can connect to Elasticsearch by pinging the server.
+     *
+     * @return If dotCMS can connect to Elasticsearch, returns {@code true}.
+     */
+    boolean canConnectToES() {
+        try {
+            RestHighLevelClientProvider.getInstance().getClient().ping(RequestOptions.DEFAULT);
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java:169]: this returns a boolean - let's check for it being true

**Review by dsilvam — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java (hunk 1 above)
Comment (jcastro-dotcms) [reply dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java]: Done!

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java (hunk 2 above)
Comment (jcastro-dotcms) [reply dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java]: Done!

**Review by dsilvam — APPROVED** [review]

**Pull request merged**